adapters/ikea/tradfri_color_light.py

```diff
@@ -0,0 +1,50 @@
+import Domoticz
+import json
+from adapters.base_adapter import Adapter
+from devices.color_light import ColorLight
+
+class TradfriColorLight(Adapter):
+    """adapter for the IKEA Tradfri color control light"""
+    def __init__(self, devices):
+        super().__init__(devices)
+        self.devices.append(ColorLight(devices, 'dimmer', 'state_brightness_color'))
+        
+    def handleMqttMessage(self, device_data, message):
```

Comment:
You could just remove this method as we have the same in Adapter which this class is inherited from